keyboards/handwired/colorlice/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+RGB_MATRIX_ENABLE = WS2812	# Enable keyboard RGB backlight functionality with ws2812 driver
```

Comment:
same. move this to top level rules.mk please.

Reply:
You can disable this attribute in the underglow variant so they don't clash.

Reply:
Moved. But could you please show me how to disable this attribute in the underglow. Thus my pcb support both Underglow and Switch LEDs. But they won't be actived at the same time. The user have to choose which option they might want.

Reply:
I'll take a look right now

Reply:
@marhalloweenvt just add a rules.mk into the default_rgb_underglow/ folder
```
RGB_MATRIX_ENABLE = no
RGBLIGHT_ENABLE = yes		# Enable keyboard RGB underglow
```
That will allow it to compile and leave the default working.